learn/wavemaker-release-notes/v10-13-2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "WaveMaker 10.13.2 - Release date: 11 April 2022"
+id: ""
+sidebar_label: "Release v10.13.2"
+---
+*WaveMaker released another version update with new features and bug fixes.*
+
+## Improvements
+---
+
+- FileUpload widget: **Clear** and **Delete** methods are introduced to clear and delete uploaded files. 
```

Comment:
FileUpload widget: A method named **clear** is introduced on the widget to clear the upload files status from the UI.

Reply:
We should add this method to widget documentation first https://docs.wavemaker.com/learn/app-development/widgets/form-widgets/file-upload